src/block_implementations/Eos/EosFaderBankBlock.h

```diff
@@ -4,17 +4,19 @@
 #include <QDateTime>
 #include "core/block_data/BlockBase.h"
 #include "eos_specific/EosOSCMessage.h"
+#include "core/block_data/OneOutputBlock.h"
 #include "utils.h"
 
 
-class EosFaderBankBlock: public BlockBase
+class EosFaderBankBlock: public OneOutputBlock
 {
     Q_OBJECT
 
     Q_PROPERTY(int page READ getPage WRITE setPageFromGui NOTIFY pageChanged)
     Q_PROPERTY(QString bankLabel READ getBankLabel NOTIFY bankLabelChanged)
     Q_PROPERTY(QList<QString> faderLabels READ getFaderLabels NOTIFY faderLabelsChanged)
     Q_PROPERTY(QList<qreal> faderLevels READ getFaderLevels NOTIFY faderLevelsChanged)
+    Q_PROPERTY(bool feedbackEnabled READ getFeedbackEnabled WRITE setFeedbackEnabled  NOTIFY feedbackEnabledChanged)
```

Comment:
The BoolAttribute already includes a Q_PROPERTY. The property here, the getter and setter methods and the signal can be removed.